solidity-plugin/src/main/java/org/sonarsource/solidity/CognitiveComplexityVisitor.java

```diff
@@ -80,30 +84,28 @@ public Token visitIfStatement(IfStatementContext ctx) {
     } else {
       nestingLevel++;
       complexity += nestingLevel;
-      System.out.println("If: " + complexity);
+      // System.out.println("If: " + complexity);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=21&issues=AWQn3a8JYwZU9vvrp9dP&open=AWQn3a8JYwZU9vvrp9dP)

[ISSUE_KEY]: AWQn3a8JYwZU9vvrp9dP